src/core/swap/swap-api.js

```diff
@@ -81,7 +81,8 @@ export async function fetchSwapQuote(
         })
         .then(
           quote => {
-            log.warn(`${pluginId} gave swap quote:`, quote)
+            const cleaned = cleanQuote(quote)
+            log.warn(`${pluginId} gave swap quote:`, cleaned)
```

Comment:
The `cleanQuote` function messes up my TypeScript branch, since `Object` is more of a Flow thing. Let put the conversion inline, and use syntax that both types systems are happy with:

```ts
            const { fromWallet, toWallet, ...request } = quote.request
            const cleaned = { ...quote, request }
            log.warn(`${pluginId} gave swap quote:`, cleaned)
```

I tested this edited version, and it makes TypeScript happy too.

Reply:
Done. I added `= quote.request ?? {}` since anyone using a new core but with old exchange plugins might get blank `request` objects.

Reply:
Excellent